src/models/fqhe13.jl

```diff
@@ -1,12 +1,47 @@
+function ITensors.space(::SiteType"FermionK", pos::Int; p=1, q=1, conserve_momentum=true)
+  if !conserve_momentum
+    return [QN("Nf", -p) => 1, QN("Nf", q - p) => 1]
+  else
+    return [
+      QN(("Nf", -p), ("NfMom", -p * pos)) => 1,
+      QN(("Nf", q - p), ("NfMom", (q - p) * pos)) => 1,
+    ]
+  end
+end
+
+# Forward all op definitions to Fermion
+function ITensors.op!(Op::ITensor, opname::OpName, ::SiteType"FermionK", s::Index...)
+  return ITensors.op!(Op, opname, SiteType("Fermion"), s...)
+end
+ITensors.has_fermion_string(::OpName"C", ::SiteType"FermionK") = true
```

Comment:
Maybe we could use:
```julia
ITensors.has_fermion_string(opname::OpName, ::SiteType"FermionK") = has_fermion_string(opname, SiteType("Fermion"))
```
so it just uses the same definitions we already have in ITensors.jl.

Reply:
Indeed, completely obvious and missed by me.